netbox_branching/models/branches.py

```diff
@@ -540,6 +540,29 @@ def provision(self, user):
                     f"ALTER TABLE {schema_table} ALTER COLUMN id SET DEFAULT nextval(%s)", [sequence_name]
                 )
 
+                # Copy the migrations table
+                main_table = f'{MAIN_SCHEMA}.django_migrations'
```

Comment:
Might be worth noting this needs to change in v0.6.0, since the other references to `MAIN_SCHEMA` are already changed in #254.